tests/test_dcrModel.py

```diff
@@ -191,16 +193,33 @@ def makeDummyVisitInfo(self, azimuth, elevation):
         lsstPressure = 73892.*u.pascal
         lsstWeather = Weather(lsstTemperature.value, lsstPressure.value, lsstHumidity)
         lsstObservatory = Observatory(lsstLon, lsstLat, lsstAlt)
-        airmass = 1.0/np.sin(elevation.asRadians())
-        era = 0.*radians  # on the meridian
-        zenithAngle = 90.*degrees - elevation
-        ra = lsstLon + np.sin(azimuth.asRadians())*zenithAngle/np.cos(lsstLat.asRadians())
-        dec = lsstLat + np.cos(azimuth.asRadians())*zenithAngle
+        loc = EarthLocation(lat=lsstLat.asDegrees()*u.degree,
+                            lon=lsstLon.asDegrees()*u.degree,
+                            height=lsstAlt*u.m)
+        airmass = 1.0/np.sin(elevation.asDegrees())
+
+        time = Time(2000., format='decimalyear', scale='tai', location=loc)
+        # Pick a random date and time within a 20-year span
+        time += 20*365*self.rng.rand()
+        altaz = SkyCoord(alt=elevation.asDegrees(), az=azimuth.asDegrees(),
```

Comment:
It might be clearer if this was an `AltAz` constructor directly rather than the more general `SkyCoord`.

Reply:
That's a good point, though an advantage of the more general `SkyCoord` is that it allows convenient conversion to RA and Dec through `skycoord.icrs`. If I construct it as an `AltAz`, that requires an extra line of `radec = altaz.transform_to(ICRS())`. The two give the same results, so I feel it's a pretty even trade-off and I am fine using either way. What's your preference?

Reply:
Okay. Leave it then.